docsets/Neovim/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "Neovim",
-    "version": "0.7.2",
+    "version": "0.8.0",
     "archive": "Neovim.tgz",
```

Comment:
You need to update this archive as well. I'll update it by copying the one at `versions/0.8.0/Neovim.tgz`. If that's not ok, please let me know.